iphone/Classes/TiUIWebView.m

```diff
@@ -290,7 +290,10 @@ -(void)loadFile:(NSString*)absolutePath
 		textEncodingName:(NSString*)textEncodingName
 	   mimeType:(NSString*)mimeType
 {
-	NSURL *requestURL = [NSURL fileURLWithPath:absolutePath];
+    static unsigned long uniqueIdentifier = 0;
+    uniqueIdentifier++;
```

Comment:
Update: Ok, so the page-token does not work here. I would still rather fix that one instead of manipulating the URL, because it will also manipulate the `url` in the `load` event that is used by many users to check certain files, so this change would break their apps.